tests/integration/security/sds_vault_flow/sds_vault_flow_test.go

```diff
@@ -85,16 +85,17 @@ func TestSdsVaultCaFlow(t *testing.T) {
 				},
 			}
 
-			// Apply the policy to the system namespace.
-			deployment := tmpl.EvaluateOrFail(t, file.AsStringOrFail(t, "testdata/global-mtls.yaml"),
+			// Apply the policy
+			deployment := tmpl.EvaluateOrFail(t, file.AsStringOrFail(t, "testdata/config.yaml"),
```

Comment:
optional, global-mtls.yaml seesm more illustrative than config.yaml right now. `vault-service-entry.yaml` might be a better name.

Reply:
The file name is intentionally made generic for extensibility: new policies may be placed in the config.yaml.